plugins/prometheus/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## 1.2.0
```

Comment:
Maybe 2.0.0 ?

Reply:
I don't think it's a breaking change, so long as the file format is the same

Reply:
I see